terraform/modules/iam_role/riemann_monitoring/role.tf

```diff
@@ -0,0 +1,44 @@
+module "riemann_monitoring" {
+  source = ".."
+
+  role_name = "${var.role_name}"
+  iam_policy = <<EOF
+{
```

Comment:
WDYT about moving this into a separate template file while we're at it? I'm tired of committing policies with invalid JSON 😄